python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -148,6 +150,18 @@ def from_xml(cls, xml: _XMLWrapper) -> "NCMECEntryUpdate":
             fingerprints={
                 x.tag: x.text for x in xml.maybe("fingerprints") if x.has_text
             },
+            feedback=[
```

Comment:
blocking; Your type says dict, your code says list!

Reply:
type is `t.List[t.Dict[str, str]]`